src/components/AddBook/index.js

```diff
@@ -82,12 +97,10 @@ function AddBook() {
           setTitleErrors(data.errors.title);
           setAuthorErrors(data.errors.author);
           setGenreErrors(data.errors.genre_id);
-          console.log(data);
+        } else {
+          setRedirect(true);
```

Comment:
I've just noticed there are some other API errors you might get that you haven't accounted for:

blurb
image
year

These three also have API validation the same as title, author and genre_id, so worth adding error messages for them exactly like you have already for title, author and genre_id.

To see what I mean, try and add a book with a description longer than 500 chars, an image URL that isn't a URL and a year above 2100